exporter/importables.go

```diff
@@ -751,4 +753,62 @@ var resourcesMap map[string]importable = map[string]importable{
 			{Path: "storage_resource_name", Resource: "azurerm_data_lake_store", Match: "name"},
 		},
 	},
+	"databricks_global_init_script": {
+		Service: "workspace",
+		Name: func(d *schema.ResourceData) string {
+			name := d.Get("name").(string)
+			if name == "" {
+				return d.Id()
+			}
+			re := regexp.MustCompile(`[^0-9A-Za-z_]`)
+			return re.ReplaceAllString(name, "_")
+		},
+		List: func(ic *importContext) error {
+			globalInitScripts, err := workspace.NewGlobalInitScriptsAPI(ic.Context, ic.Client).List()
+			if err != nil {
+				return err
+			}
+			for offset, gis := range globalInitScripts {
+				ic.Emit(&resource{
+					Resource: "databricks_global_init_script",
+					ID:       gis.ScriptID,
+				})
+				log.Printf("[INFO] Scanned %d of %d clusters", offset+1, len(globalInitScripts))
+			}
+			return nil
+		},
+		Body: func(ic *importContext, body *hclwrite.Body, r *resource) error {
+			gis, err := workspace.NewGlobalInitScriptsAPI(ic.Context, ic.Client).Get(r.ID)
+			if err != nil {
+				return err
+			}
+			err = os.Mkdir(fmt.Sprintf("%s/global-init-scripts", ic.Directory), 0755)
+			if err != nil && !os.IsExist(err) {
+				return err
+			}
+			local, err := os.Create(fmt.Sprintf("%s/global-init-scripts/%s", ic.Directory, path.Base(r.Name)))
```

Comment:
can't we have it in `files`, which hold existing init scripts as well?..

Reply:
ok, I first thought about putting it there, then thought about separate place